packaging/install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -115,4 +115,4 @@ spec:
               cpu: 200m
               memory: 384Mi
   strategy:
-    type: Recreate
+    type: Recreate
```

Comment:
Why did you changed this?

Reply:
Ahh it might have changed when I was setting my docker images. Sorry